### DISCUSSION TITLE
Do not allow users to bypass malicious website strict blocking

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2943] [Ideas] [answered]
uBlock Origin should remove the Proceed option for websites blocked by uBlock badware risk. It can prevent any people who don't know what they are doing (like children, old people) to accidentally clicked the button and visit a malicious websites. People who know what they are doing can still disable uBO or add a exception rule to visit the website.

**Comment by Yuki2718** [comment, marked as answer]
There can always be false flag and it actually happend. This also requires uBO to distinguish the blocking lists after strict-block and then change UI (currently it doesn't distinguish but only looks up in which list the blocking rule belongs to).

**Comment by iam-py-test** [comment]
What about an admin setting to disable any bypassing of strict blocking?
Though, it's largely pointless as a user can easily disable/uninstall uBo, disable filterlists, etc.
Thanks

**Comment by DandelionSprout** [comment]
I admit I have first-hand been involved in various false positives over the years, both in my lists and in others' lists.

One example was a year or two ago, when it was only through list user feedback that I became aware that e.g. `||discord*.gift^$all` was in fact blocking Discord's own sole real gift domain `discord.gift`.
